types/queries/word_by_id.sql

```diff
@@ -0,0 +1 @@
+select * from word where id = $1
```

Comment:
Use the existing query for word by id, I think we have one already. If not, then use the BasicWord struct and query for specific columns here. I try to avoid `select *`

Reply:
If you need this query, then only select the field you need and make the name more specific. "document_id_for_word" or some such